<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortageBase</class>
 <widget class="QWidget" name="PortageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>502</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>&lt;font size=&quot;+1&quot;&gt;Packages in Portage&lt;/font&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <property name="indent">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="spacer13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="KPushButton" name="pbWhatsThis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>What's This</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout">
     <item row="0" column="4">
      <widget class="QLineEdit" name="searchFilter">
       <property name="toolTip">
        <string>Filter packages here...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>35</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="textLabel1_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Filter:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="KButtonGroup" name="filterGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioAll">
          <property name="text">
           <string>&amp;All Packages</string>
          </property>
          <property name="shortcut">
           <string>Alt+A</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="buttonGroupId" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioUpdates">
          <property name="text">
           <string>&amp;Updates</string>
          </property>
          <property name="shortcut">
           <string>Alt+U</string>
          </property>
          <property name="buttonGroupId" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioInstalled">
          <property name="text">
           <string>I&amp;nstalled Packages</string>
          </property>
          <property name="shortcut">
           <string>Alt+N</string>
          </property>
          <property name="buttonGroupId" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="KPushButton" name="pbClearFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear filter</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSplitter" name="splitterH">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="CategoriesListView" name="categoriesView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SubCategoriesListView" name="subcategoriesView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layout18">
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QSplitter" name="splitterV">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="PortageListView2" name="packagesView2"/>
         <widget class="QFrame" name="portageFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32767</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QGridLayout">
           <item row="0" column="0">
            <widget class="KTextBrowser" name="summaryBrowser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout">
             <item row="0" column="2">
              <spacer name="spacer9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="KPushButton" name="pbUninstall">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Uninstall selected packages...</string>
               </property>
               <property name="text">
                <string>Uninstall</string>
               </property>
               <property name="shortcut">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="KPushButton" name="pbQueue">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Add/Remove selected packages to the Installation Queue</string>
               </property>
               <property name="text">
                <string>Add to &amp;Queue</string>
               </property>
               <property name="shortcut">
                <string>Alt+Q</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="KPushButton" name="pbAdvanced">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Show package details...</string>
               </property>
               <property name="text">
                <string>Details...</string>
               </property>
               <property name="shortcut">
                <string>Return</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>CategoriesListView</class>
   <extends>QWidget</extends>
   <header>categorieslistview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubCategoriesListView</class>
   <extends>QWidget</extends>
   <header>categorieslistview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PortageListView2</class>
   <extends>QTreeView</extends>
   <header>portagelistview2.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">categorieslistview.h</include>
  <include location="local">categorieslistview.h</include>
  <include location="local">portagelistview.h</include>
  <include location="local">ktextbrowser.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
